### DISCUSSION TITLE
Welcome to ReactSelfbot Discussions!

### DISCUSSION THREAD

**Discussion by cranberry3148** [FlawCra/ReactSelfbot#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by NightOwl0002** [comment]
Hiya! Where do we go to create an account to login to react?

**Reply by cranberry3148** [reply]
the authentication was removed serverside so you should be able to just enter anything you want
